src/map/js/components/LayerPanel/FireHistoryTimeline.js

```diff
@@ -28,16 +28,29 @@ export default class FireHistoryTimeline extends React.Component {
 		}
 	}
 
+	increaseFireHistoryYear = (evt) => {
+		this.props.fireHistorySelectIndex < 14 ? layerActions.incrementFireHistoryYear() : '';
```

Comment:
@dainguyendo Using a ternary here is not optimal. A simple 'if' statement is a much cleaner solution, and eslint will tell you the same because if 'fireHistorySelectIndex' is Not less than 14 you are evaluating to an empty string, which doesn't make sense to someone reading this w/ no context.